service/slice_config_webhook_validation_test.go

```diff
@@ -1420,7 +1426,135 @@ func ValidateMaxClusterForParticipatingCluster(t *testing.T) {
 	require.Contains(t, err.Error(), "participating clusters cannot be greater than MaxClusterCount")
 	clientMock.AssertExpectations(t)
 }
+func TestValidateCertsRotationInterval_Postive(t *testing.T) {
```

Comment:
Earlier I had given comments to fix the spelling of postive TestValidateCertsRotationInterval_Postive?